Python/pywarpx/fields.py

```diff
@@ -508,6 +508,20 @@ def _setitem2d(self, index, value):
                     fields[i][sss] = value
 
 
+def rhoWrapper(level=0, include_ghosts=False):
```

Comment:
This routine is not needed since there is already `RhoFPWrapper` which does the same thing. That also uses the consistent naming, since it is fetching `rho_fp`.

Reply:
Thanks! I completely missed that `RhoFPWrapper` was already there.